src/app/pipelines-dashboard/pipelines-dashboard.component.html

```diff
@@ -2,7 +2,7 @@
   <div class="rounded border px-2 py-2 bg-white my-3 stop-scroll">
     <div class="row">
       <div class="col-6 my-auto">
-        <h5 class="m-0 p-0">OpenEBS {{Engine}} Pipelines</h5>
+        <h5 class="m-0 p-0">OpenEBS {{getBranchName(Engine)}} Pipelines</h5>
```

Comment:
Are we passing variables as capital case ?

Reply:
no , variable names will be in camelCase . 
Updated 👍